virttest/env_process.py

```diff
@@ -1227,6 +1227,17 @@ def preprocess(test, params, env):
     LOG.debug("KVM userspace version(qemu): %s", kvm_userspace_version)
     version_info["qemu_version"] = str(kvm_userspace_version)
 
+    #Get the SLOF version for power pc
+    slof_ver_cmd = params.get("slof_ver_cmd", "rpm -qa|grep SLOF")
+    if arch.ARCH in ('ppc64', 'ppc64le'):
+        if slof_ver_cmd:
+            try:
+                slof_version = a_process.run(
+                    slof_ver_cmd, shell=True).stdout_text.strip()
+                LOG.debug("SLOF version: %s", slof_version)
```

Comment:
Save to `version_info["firmware_version"]`

Reply:
Updated, thanks.